scripts/ci/run-ci-tests.sh

```diff
@@ -238,12 +238,7 @@ if [ -n "$TRAVIS" ] || [ -n "$CIRCLECI" ]; then
        # Error (criu/tty.c:1014): tty: Don't have tty to inherit session from, aborting
        make -C test/others/shell-job/ run
 fi
-
-# FIXME: rpc tests fail even with set glibc tunable
-# https://github.com/checkpoint-restore/criu/issues/1696
-if [ "$GLIBC_TUNABLES" != "glibc.pthread.rseq=0" ]; then
-	make -C test/others/rpc/ run
-fi
+make -C test/others/rpc/ run
```

Comment:
We also need to revert https://github.com/checkpoint-restore/criu/commit/f008f740411156b7309219c7e052cb8bc24d5aae

Reply:
thanks!